app/register/page.tsx

```diff
@@ -1,19 +1,48 @@
 "use client";
 import React, { useState } from "react";
-import { Button, TextField, Typography, Container, Grid } from "@mui/material";
+import {
+  Button,
+  TextField,
+  Typography,
+  Container,
+  Grid,
+  FormControlLabel,
+  Checkbox
+} from "@mui/material";
 import { postData, postRegsiterData } from "@/services/api";
+import {
+  button,
+  headerDetails,
+  section1,
+  section2,
+  section3,
+  section4,
+  section5,
+  section6,
+  section7,
+  section8,
+  section9,
+  section10
+} from "@/data/registerPage";
+import Box from "@mui/material/Box";
+import ArrowBack from "@mui/icons-material/ArrowBack";
 
 interface RegisterData {
-  username: string;
```

Comment:
Why is the username removed? Keep it even if it's not used in the updated FE

Reply:
There are two names, first name and last name, so change it that way Akka.